validate/BUILD

```diff
@@ -41,7 +42,19 @@ cc_library(
     hdrs = ["validate.h"],
 )
 
+go_library(
+    name = "validate",
+    embed = [":validate_go_proto"],
+    importpath = "github.com/envoyproxy/protoc-gen-validate/validate",
+)
+
```

Comment:
Can we call this `validate_go` for consistency?

Reply:
I suppose that would be consistent with the other targets in this file.  However, Gazelle expects a target named `validate`, not `validate_go`.

Reply:
What about `validate_go` and an alias rule with a note that 'this is needed because Gazelle is overly opinionated'? I'm not looking forward to the day when we incorporate some other tool that requires that the default target be the Python version or something.

Reply:
This makes sense.